### DISCUSSION TITLE
Compiling Error: import_error: module 'persist' not found - solidify_all.be

### DISCUSSION THREAD

**Discussion by wjohn007** [arendst/Tasmota#21866] [Q&A]
I am just dealing with berry-code to be solidified.
On of my classes uses 'import persist' which causes following error

`Parsing:  B01BaseClass.be
import_error: module 'persist' not found`

I assume the the term 'persist' is missed in file solidify_all.be

`var globs = "path,ctypes_bytes_dyn,tasmota,ccronexpr,gpio,light,webclient,load,MD5,lv,light_state,udp,tcpclientasync,"
            "lv_clock,lv_clock_icon,lv_signal_arcs,lv_signal_bars,lv_wifi_arcs_icon,lv_wifi_arcs,"
            "lv_wifi_bars_icon,lv_wifi_bars,"
            "_lvgl,"
            "int64"`

Am I right ?

Addendum: 'weberserver' is also missing

**Comment by s-hadinger** [comment]
This is weird. This error means that `import persist` is actually executed, which does not make sense in the solidification process. If your code contains `import persist`, it is compiled as such but does not require the module to exist at compile time.

Indeed `webserver` is certainly missing
